CHANGELOG.md

```diff
@@ -1,5 +1,67 @@
 # Changelog
 
+## v0.111.0
+
+## Unreleased
+
+This Splunk OpenTelemetry Collector release includes changes from the opentelemetry-collector v0.111.0 and the opentelemetry-collector-contrib v0.111.0 releases where appropriate.
+
+### 🛑 Breaking changes 🛑
+
+
+- (Core) componentstatus: Remove deprecated functions ([#11363](https://github.com/open-telemetry/opentelemetry-collector/pull/11363))
+
+- (Core) service: remove deprecated types ([#11361](https://github.com/open-telemetry/opentelemetry-collector/pull/11361))
```

Comment:
We don't need to include any changes from API changelog, only changes from End-user changelog https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.111.0